### PR TITLE
Add hook and fix hook templates.table-row

### PR DESCRIPTION
**Pull request by gillesbourgeat**
Hello,

Add new hook : 
- profile.table-row
- profile.table-header
- import.table-row
- import.table-header
- export.table-row
- export.table-header

Add parameter template_id for hook templates.table-row in backoffice -> templates.html

cheers,

### CODE REVIEW

**Comment by lovenunu** [comment]
You don't need to use the ```location``` parameter when you write a new hook. It was used for backward compatibility of 2.0 modules ```AdminIncludes```.

but anyway, good job ;)

File: templates/backOffice/default/profiles.html [hunk 1]
```diff
@@ -75,6 +76,7 @@
                                         {/if}
                                     </td>
                                     <td>{$DESCRIPTION}</td>
+                                    {hook name="profile.table-row" location="profile_table_row" sale_id=$ID}
```
Comment (bibich) [inline templates/backOffice/default/profiles.html]: It should be `profile_id` ?

**Comment by gillesbourgeat** [comment]
@bibich :+1: 
@lovenunu :+1: 

This pull request will come back.

Terminator

**Comment by lovenunu** [comment]
Why did you close your PR ? O.o

**Comment by gillesbourgeat** [comment]
The better to reopen later.

**Pull request closed without merging**